doc/descriptors.md

```diff
@@ -73,6 +73,7 @@ Descriptors consist of several types of expressions. The top level expression is
 - `tr(KEY)` or `tr(KEY,TREE)` (top level only): P2TR output with the specified key as internal key, and optionally a tree of script paths.
 - `addr(ADDR)` (top level only): the script which ADDR expands to.
 - `raw(HEX)` (top level only): the script whose hex encoding is HEX.
+- `rawtr(KEY)` (top level only): P2TR output with the specified key as tweaked key.
```

Comment:
I think saying `tweaked key` could be confusing unless you know what's going on under the hood of taproot. Maybe something like `output key` would be better

Reply:
Agree.

Reply:
Or maybe call it pre-tweaked. I can see how that's more useful than a raw descriptor with the tweaked key.

Reply:
Done, renamed to "output key".